docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md

```diff
@@ -81,6 +81,13 @@ $ helm version
 
 The output should be similar to the following:
 
-```
+```output
 version.BuildInfo{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"}
 ```
+
+You can use Helm to configure the CPU and memory allocation for each Yugabyte Platform container. For example, the following command sets the container to 3 GB and Prometheus CPU to 3 cores:
```

Comment:
this should ideally go under https://docs.yugabyte.com/latest/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes/#customization

Reply:
Thanks for the suggestion. I checked the page, and this customization is already described there.
I deleted the content from install-yugabyte-platform/prepare-environment/kubernetes.md and cleaned up install-yugabyte-platform/install-software/kubernetes/#customization.